tests/resources/openassetio-traitgen-test-all.yaml

```diff
@@ -33,9 +33,6 @@ traits:
                     boolProperty:
                         type: boolean
                         description: A bool-typed property.
-                    dictProperty:
```

Comment:
[fi] Can we add a todo/comment out instead?

Reply:
Added a TODO in ccacc968